### PR TITLE
Bayu/feat/crop image

### PR DESCRIPTION
**Pull request by aliffarisqi**
# Description: 
- add cropper feature

# Changes: 
- Added fontawesome package, corpper image package

# Notes/Blocker:
- jadi kalau di android tampilannya ga see customize di ios
- 
# Preview: iOS

WhatsApp Image 2024-07-18 at 21 57 38
WhatsApp Image 2024-07-18 at 21 57 39
WhatsApp Image 2024-07-18 at 21 57 39 (1)

# Preview: Android
Screenshot 2024-07-18 at 21 59 06
Screenshot 2024-07-18 at 21 59 23

### CODE REVIEW

File: lib/Core/Widget/button/camera_button.dart
Comment (gregoriusyuristama) [inline lib/Core/Widget/button/camera_button.dart:1]: pindah ke `Features/Capture/Components` aja kalo komponennya cuma dipake di 1 fitur

File: lib/Core/Widget/button/flash_button.dart
Comment (gregoriusyuristama) [inline lib/Core/Widget/button/flash_button.dart:1]: same as above

File: lib/Core/Widget/button/galery_button.dart
Comment (gregoriusyuristama) [inline lib/Core/Widget/button/galery_button.dart:1]: same as above

File: lib/Core/Widget/button/notebook_button.dart
Comment (gregoriusyuristama) [inline lib/Core/Widget/button/notebook_button.dart:1]: same as above

File: android/app/src/main/AndroidManifest.xml
Comment (gregoriusyuristama) [inline android/app/src/main/AndroidManifest.xml:1]: yang ios udah ada permissionnya ya? 🤔

**Review by gregoriusyuristama — APPROVED** [review]
approve with comment, after resolving comment bisa langsung squash and merge

**Comment by gregoriusyuristama** [comment]
> 2 commit terakhir perubahan terbaru, btw kenapa commit lama ngikut terus ya ?

pastiin bikin branchnya dari main, jangan dari branch lama lu bay.

- Flow nya after pull request di merge update main local > bikin branch baru
- Kalo lagi in progress bikin fitur di sebuah branch terus ada merge an baru better choicenya bisa rebase branch lu sekarang sebelum bikin pull request, kalo masih blm paham merge main terbaru dulu gpp. Kalo bingung bisa tanya gw

**Comment by aliffarisqi** [comment]
oke cho, iya aku kalau bikin branch checkout dari current branch, oke untuk nextnya berarti dari main ya, 
> main local ku nge pull dari main remote, terus buat branch baru dari situ kan ya

**Pull request merged**
